terraform/modules/scheduled-function/variables.tf

```diff
@@ -97,6 +97,12 @@ variable "min_instance_count" {
   default     = 1
 }
 
+variable "retries_enabled" {
```

Comment:
This will be very useful to disable for many cronjobs!  Where if it fails we just wait for the next run again 5 minutes later.